### ISSUE TITLE
Hey!

### ISSUE THREAD

**Issue by jwanglof** [erwan-lemonnier/cloud-storage-image-resizer#1]
Hey!
Awesome library, will use it for a website since we're migrating to Google Cloud with Google Cloud Storage :) 

Since this library is almost what we need but not quite, I was wondering if it's OK to create a PR and contribute? :)

Cheers,
Johan

**Comment by erwan-lemonnier** [comment]
Hi Johan!

Sure! Shoot ahead!
I built it in a rush for a recent project (gofrendly.com) so it’s not a clean job by far, and lacks a lot of features: feel free to go wild. We could even discuss changing the semantics.

I am going to add a few things myself in the coming days, in particular an option to load an image from a bytestring rather than a url in the fetch method.

/Erwan

**Comment by jwanglof** [comment]
Just opened https://github.com/erwan-lemonnier/cloud-storage-image-resizer/pull/2 with my changes :)

**Comment by erwan-lemonnier** [comment]
Merged! Thanks Johan!
I took the liberty to make a few additional edits:
- renamed '__base_image' to '__load_image' (since it is more explicit of
what it does)
- renamed 'store' to 'store_and_return_url' (more consistent with
store_and_return_blob)
- added load_from_bytestring() which I need for my current project
- added your name to the README (if it's ok with you?)

I couldn't merge your last pull request (#3) so I guess my latest push will
mess things up for you. Sorry...

This first merge request is merged and released on pypi:
google-cloud-storage-image-resizer-0.0.18.tar.gz

Stor tack för din hjälp ;-)
/Erwan
